modules/gapi/perf/cpu/gapi_render_perf_tests_cpu.cpp

```diff
@@ -0,0 +1,59 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+
+#include "../perf_precomp.hpp"
+#include "../common/gapi_render_perf_tests.hpp"
+#include <opencv2/gapi/render.hpp>
+
+#define RENDER_CPU cv::gapi::render::ocv::kernels()
+
+namespace opencv_test
+{
+
+
+INSTANTIATE_TEST_CASE_P(RenderTestTextsCPU_8U, RenderTestTexts,
```

Comment:
What about NV12 instances ?

Reply:
We can use a common instance for NV12 and BGR tests cases, aren't we?

Reply:
Yes, could you fix it for unit tests as well, please ?